scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -10,10 +10,11 @@ workflow GvsBulkIngestGenomes {
     input {
         # Begin GvsPrepareBulkImport
         # for now set the entity type names with a default
-        String samples_table_name = "sample"
-        String sample_id_column_name = "sample_id"
+        String data_table_name = "sample" ## Note that it is possible an advanced user has a different name for the table. We could glean some information from the sample_set name if that has been defined, but this has not, and try to use that information instead of simply using the default "sample"
```

Comment:
Is the "We could glean..." something that's implemented here or a TODO?

Reply:
it's implemented in the python

Reply:
but we dont really use it cuz it's a given